### DISCUSSION TITLE
Will json and x.json2 always coexist?

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#21842] [Questions and Answers] [answered]
Will x.json2 replace json in the future?

**Comment by medvednikov** [comment]
No. The pure V version will become the one.

**Reply by JalonSolov** [reply, marked as answer]
That means yes (to the question in the first comment, not the title of the discussion), since `x.json2` _is_ the pure V version, while `json` is the wrapper on cJSON.

So `x.json2` will replace `json` in the future.  They will not always coexist.
